.gitignore

```diff
@@ -2,4 +2,17 @@
 /notes
 
 # Spellchecker results file
-Spellchecker.results
\ No newline at end of file
+Spellchecker.results
+
+# Latex generated files
+/docs/tex/*.aux
```

Comment:
Since I don't know yet what language and tools I'll be using I'd rather have them, though these extensions do not seem to be something that should be on a repo anyways.

I'll leave it as it is for now, and probably tidy it up later when I start development.

Reply:
extensions like `*.aux` are OK. It's the path what's not.